openfisca_uk/variables/wra/land_transaction_tax.py

```diff
@@ -0,0 +1,94 @@
+from openfisca_uk.model_api import *
+
+
+class ltt_liable(Variable):
+    label = "Liable for Land Transaction Tax"
+    documentation = (
+        "Whether the household is liable to pay the Land Transaction Tax"
+    )
+    entity = Household
+    definition_period = YEAR
+    value_type = bool
+
+    def formula(household, period):
+        country = household("country", period)
+        return country == country.possible_values.WALES
+
+
+class ltt_on_residential_property(Variable):
+    label = "LTT on residential property"
```

Comment:
This PR is inconsistent on acronyms in labels vs documentation. I'd prefer keeping acronyms out of labels and putting them inside parens in the documentation. I'd also suggest putting the country in the documentation.